code/tests/utilities/helpers/test_ConfigHelper.py

```diff
@@ -140,6 +155,54 @@ def test_default_config_is_cached():
     assert default_config_one is default_config_two
 
 
+def test_default_config_when_use_advanced_image_processing():
+    # given
+    env_helper_mock.USE_ADVANCED_IMAGE_PROCESSING = True
```

Comment:
```suggestion
def test_default_config_when_use_advanced_image_processing(env_helper_mock):
    # given
    env_helper_mock.USE_ADVANCED_IMAGE_PROCESSING = True
```

Reply:
Nah the naming is fine. I just wasn't paying enough attention, good catch!